common/models/workspace.js

```diff
@@ -33,6 +33,15 @@ module.exports = function(Workspace) {
     var ViewDefinition = app.models.ViewDefinition;
     var TEMPLATE_DIR = path.join(__dirname, '..', '..', 'templates', 'projects');
     var DEFAULT_TEMPLATE = 'api-server';
+    var DEFAULT_LB_VERSION = '2.x';
+    var DEPENDENCY_3_X = {
+      'loopback': '^3.0.0-alpha.1',
+      'loopback-datasource-juggler': '^3.0.0-alpha.1',
```

Comment:
In 3.0, juggler is a regular dependency of loopback, it should not be saves in application dependencies.